src/Lender.sol

```diff
@@ -670,16 +672,19 @@ contract Lender {
     // @param swapMinimum: The minimum amount of lst to receive
     // @returns lent: The amount of underlying to be lent
     // @returns slippageRatio: The slippageRatio of the swap (1e18 based % to adjust swivel orders if necessary)
-    function swapETH(address lst, uint256 amount, uint256 swapMinimum) internal returns (uint256 lent, uint256 slippageRatio) {
-        payable(ETHWrapper).transfer(amount);
-        if (lst != address(0)) {
-            (lent, slippageRatio)  = IETHWrapper(ETHWrapper).swap(WETH, lst, amount, swapMinimum);
-            return (lent, slippageRatio);
-        }
-        // If the `lst` address is blank, no swap is necessary
-        else {
-            return (amount, 1);
+    function swapETH(address pool, address lst, uint256 amount, uint256 swapMinimum) internal returns (uint256 lent, uint256 slippageRatio) {
+        // Conduct the lend operation to acquire principal tokens
+        (bool success, bytes memory returndata) = ETHWrapper.delegatecall(
+            abi.encodeWithSignature('swap(address,address,address,uint256,uint256)', pool, "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE", lst, amount, swapMinimum));
```

Comment:
Why are we using `"0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"` instead of `Lender.WETH` here?

Reply:
Good call -- (it was because lender.WETH didnt exist before this PR woops)